### DISCUSSION TITLE
Add Chuck McAndrew to the Core team

### DISCUSSION THREAD

**Discussion by pavithraes** [nebari-dev/nebari#2119] [Community]
@dcmcand has contributed to several Nebari repos and regularly participated in the Nebari community meetings for the past couple of months. He will also be managing Nebari releases and help with maintenance moving forward. I nominate Chuck to the Nebari Core team to support his work. :)

**Comment by pavithraes** [comment]
@nebari-dev/core-team Please share any thoughts, and add Chuck if there are no objections. :)

**Reply by costrouc** [reply]
I have no issues with adding @dcmcand. Thanks @pavithraes for opening this

**Comment by dharhas** [comment]
Added @dcmcand and removed @iameskild

**Reply by pmeier** [reply]
xref to #2160 for why @iameskild was removed.
